me.broken_by.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,35 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - pmedit-cli
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v app/* ${FLATPAK_DEST}
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
+      - cp -v src/flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
+      - cp -v src/flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
+      - cp -v src/flatpak/me.broken_by.PdfMetadataEditor.desktop ${FLATPAK_DEST}/share/applications/
+      - cp -v me.broken_by.PdfMetadataEditor.metainfo.xml ${FLATPAK_DEST}/share/metainfo/
```

Comment:
I suggested changes here that you discarded.

Reply:
Yes, sorry about that. Added them.